### PR TITLE
Bug/rain 3142 Fixed owner history modal design as per figma

### PR DESCRIPTION
**Pull request by ayan-egov**

### CODE REVIEW

**Review by harit-egov — APPROVED** [review]

File: packages/modules/pt/src/pages/citizen/MyProperties/TransferDetails.js [hunk 1]
```diff
@@ -2,25 +2,25 @@ import React from "react";
...
       </div>
     );
   });
 };
 
-const TransferDetails = ({ data }) => {
+const TransferDetails = ({ data, showHorizontalBar, wrapperStyles, tableStyles, containerStyles, rowContainerStyles }) => {
   return (
-    <div style={{ borderLeft: "2px solid rgba(0, 0, 0, 0.12)", marginTop: "-5px", paddingLeft: "5px", paddingTop: "10px" }}>
+    <div style={wrapperStyles}>
```
Comment (saurabh-egov) [inline packages/modules/pt/src/pages/citizen/MyProperties/TransferDetails.js:18]: use css

File: packages/modules/pt/src/pages/citizen/MyProperties/TransferDetails.js [hunk 2]
```diff
@@ -2,25 +2,25 @@ import React from "react";
...
       {data.map((value, index) => {
         return (
-          <div>
-            {index !== 0 && <div style={{ backgroundColor: "rgba(0, 0, 0, 0.12)", width: "auto", height: "2px", marginLeft: "16px" }}></div>}
-            <StatusTable>{returnTransferData(value)}</StatusTable>
+          <div style={containerStyles}>
```
Comment (saurabh-egov) [inline packages/modules/pt/src/pages/citizen/MyProperties/TransferDetails.js:21]: use css

File: packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js [hunk 3]
```diff
@@ -82,6 +82,86 @@ const propertyOwnerHistory = ({ userType, propertyId: propertyIdFromProp }) => {
...
+      <React.Fragment>
+        <Card>
+          <div>
+            {Object.keys(ownershipInfo).map((key, index, arr) => {
+              const date = convertEpochToDate(Number(key));
+              return (
+                <div style={{ margin: "0 10px", position: "relative", paddingBottom: "5px" }}>
+                  <div style={{ display: "flex" }}>
+                    <div
+                      style={{
```
Comment (saurabh-egov) [inline packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js:97]: use css

File: packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js [hunk 4]
```diff
@@ -82,6 +82,86 @@ const propertyOwnerHistory = ({ userType, propertyId: propertyIdFromProp }) => {
...
+                        display: "inline-block",
+                        position: "relative",
+                        left: "-9px",
+                        top: "4px",
+                        zIndex: 10,
+                      }}
+                    ></div>
+                    <div style={{ display: "flex", flexDirection: "column" }}>
+                      <CardSubHeader
+                        style={{
```
Comment (saurabh-egov) [inline packages/modules/pt/src/pages/citizen/MyProperties/propertyOwnerHistory.js:111]: use css

**Review by saurabh-egov — APPROVED** [review]

**Pull request merged**